From ef82b9247d34bb118b7280ceb003c65157b43ef0 Mon Sep 17 00:00:00 2001
From: Scott D Phillips <scott.d.phillips@intel.com>
Date: Wed, 14 May 2014 16:19:44 +0000
Subject: [PATCH] HACK: sdhci-acpi: add SDHCI_QUIRK2_BROKEN_HS200 to Intel
 Baytrail-T platform

Disable HS200 which some boards are having trouble with.

Change-Id: Ifcb4b27e27562d4cbe2a87051d562685ed5e6fb0
Signed-off-by: Scott D Phillips <scott.d.phillips@intel.com>
---
 drivers/mmc/host/sdhci-acpi.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/sdhci-acpi.c b/drivers/mmc/host/sdhci-acpi.c
index b6f4c6f..0a7ce45b 100644
--- a/drivers/mmc/host/sdhci-acpi.c
+++ b/drivers/mmc/host/sdhci-acpi.c
@@ -119,7 +119,7 @@ static const struct sdhci_acpi_slot sdhci_acpi_slot_int_emmc = {
 	.caps    = MMC_CAP_8_BIT_DATA | MMC_CAP_NONREMOVABLE | MMC_CAP_HW_RESET,
 	.caps2   = MMC_CAP2_HC_ERASE_SZ | MMC_CAP2_POLL_R1B_BUSY,
 	.flags   = SDHCI_ACPI_RUNTIME_PM,
-	.quirks2 = SDHCI_QUIRK2_TUNING_POLL,
+	.quirks2 = SDHCI_QUIRK2_TUNING_POLL | SDHCI_QUIRK2_BROKEN_HS200,
 };
 
 static const struct sdhci_acpi_slot sdhci_acpi_slot_int_sdio = {
-- 
1.7.9.5

